contracts/Outcome2.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.5.11;
```

Comment:
Should this file be here?

Reply:
It will disappear if #79 is merged into this branch.